frontend/src/pages/FAQs/FaqsPageData.tsx

```diff
@@ -75,93 +93,438 @@ function AnswerWhatAreHealthDisparities() {
 
 function AnswerWhatDataSources() {
   return (
-    <div className='text-small'>
-      <p>
-        In this tracker, we are using many sources, including{' '}
-        <a href={urlMap.acs5}>American Community Survey 5-year estimates</a>,
-        multiple datasets from the CDC including the{' '}
-        <a href={urlMap.cdcBrfss}>CDC’s BRFSS data set</a> via America's Health
-        Rankings, and many more. Some sources are updated bi-weekly, while other
-        important data (such as information around social determinants of
-        health) can lag from weeks to years. Specific information on update
-        frequencies by source can be found on our{' '}
-        <a href={DATA_CATALOG_PAGE_LINK}>Data Downloads</a> page.
+    <>
+      <p>
+        In this tracker, we use a variety of reputable data sources, including:
       </p>
-    </div>
+      <p className='my-0 py-0'>
+        <span className='my-0 py-0'>
+          <IndentedItem
+            label={`American Community Survey 5-year estimates`}
+            pClassName='inline mr-1'
+          />
+          <HetLaunchLink
+            label={'American Community Survey 5-year estimates'}
+            href={'urlMap.acs5'}
+          />
+          ,
+        </span>
+      </p>
+      <p className='my-0 py-0'>
+        <span className='my-0 py-0'>
+          <IndentedItem
+            label={`multiple datasets from the CDC, such as the CDC’s Behavioral Risk Factor Surveillance System (BRFSS) data set`}
+            pClassName='inline mr-1'
+          />
+          <HetLaunchLink
+            label={`CDC’s BRFSS data set`}
+            href={urlMap.cdcBrfss}
+          />
+          ,
+        </span>
+      </p>
+      <IndentedItem
+        label={`sourced via America's Health
+        Rankings, among others.`}
+        pClassName='inline'
+      />
```

Comment:
i think this can be cleaned up; is there a way to avoid having to use IndentedItem inside a span inside a p? Also, it renders strangely; i think it should either be a bulleted list or a paragraph type list, but right now it reads like a paragraph split up by indented line breaks
<img width="700" alt="Screenshot 2024-11-05 at 8 41 33 AM" src="https://github.com/user-attachments/assets/4c4fc8b4-891c-4314-8d35-ae19bd5d7e1e">